downstream/modules/platform/proc-create-keycloak-realm.adoc

```diff
@@ -0,0 +1,48 @@
+[id="proc-create-keycloak-realm_{context}"]
+
+= Creating a Keycloak Realm for {PlatformNameShort}
+
+Use the following procedure to create the Keycloak Realm.
+
+.Procedure
+. Navigate to menu:Operator[Installed Operators].
+. Select the *Red Hat Single Sign-On Operator* project.
+. Select the *Keycloak Realm* tab and click btn:[Create Keycloak Realm].
+. On the Keycloak Realm form, select btn:[YAML view].
+Edit the YAML file as follows:
++
+[options="nowrap" subs="+quotes"]
+----
+kind: KeycloakRealm
+apiVersion: keycloak.org/v1alpha1
+metadata:
+  name: ansible-automation-platform-keycloakrealm
+  namespace: rh-sso
+  labels:
+    app: sso
+    realm: ansible-automation-platform
+spec:
+  realm:
+    id: ansible-automation-platform
+    realm: ansible-automation-platform
+    enabled: true
+    displayName: {PlatformNameShort}
+  instanceSelector:
+    matchLabels:
+      app: sso
+----
++
+
+[cols="30% 30% 30%",options="header]
+|====
+| *Field* | *Value* | *Description*
+| metadata | name: | Set a unique value in metadata for the name of the configuration resource (CR).
```

Comment:
`metadata.name` is the field. This applies to all the fields in this table. `spec.realm.id` and so on.